### PR TITLE
Css tests at rules

### PR DESCRIPTION
**Pull request by evilebottnawi**
Fix and tests. Basic comments tested around 100%.

### CODE REVIEW

**Comment by j-f1** [comment]
@evilebottnawi Can you remove the “Add Facebook to users” commit?

**Comment by evilebottnawi** [comment]
@j-f1 yep, right now

**Comment by evilebottnawi** [comment]
@j-f1 done :+1:

**Comment by evilebottnawi** [comment]
@j-f1 @lydell friendly ping

**Comment by evilebottnawi** [comment]
@j-f1 @lydell double friendly ping

File: tests/css_case/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -349,7 +349,7 @@ $KeepScssVar: val;
   }
 
   @KeepDetachedRuleset();
-  @KeepDetachedRuleset();
+  @keepdetachedruleset (/* comment */);
```
Comment (lydell) [inline tests/css_case/__snapshots__/jsfmt.spec.js.snap]: This shouldn't have been lowercased, should it?

Comment (evilebottnawi) [reply tests/css_case/__snapshots__/jsfmt.spec.js.snap]: @lydell good catch, thanks!

**Comment by evilebottnawi** [comment]
@lydell @j-f1 i remove `@KeepDetachedRuleset ( /* comment */  );` because it is invalid Less syntax for all version. Compilation error in 1, 2 and 3 version, no characters allow in parens

**Pull request merged**